Compiler/BackEnd/BackendDAEOptimize.mo

```diff
@@ -1193,7 +1193,7 @@ algorithm
       BackendDAE.Var v;
       BackendDAE.Variables vars,vars1;
       DAE.ComponentRef cr;
-    case (v as BackendDAE.VAR(varKind = BackendDAE.PARAM()),(vars,vars1))
+    case (v as BackendDAE.VAR(varKind = BackendDAE.PARAM()),(_,_))
```

Comment:
@adrpo would it make sense to run the script again to have the last match item convert from "(_,_)" to "_"?